packages/one-app-dev-bundler/index.js

```diff
@@ -14,6 +14,5 @@
  * permissions and limitations under the License.
  */
 
-import devBuildModule from './utils/dev-build-module.js';
-
-export default devBuildModule;
+export { devBuildModule } from './utils/dev-build-module.js';
+export { bundleExternalFallbacks } from './utils/bundle-external-fallbacks.js';
```

Comment:
this still needs to provide default export otherwise its considered a breaking change